eng/pipelines/templates/stages/archetype-android-release.yml

```diff
@@ -114,13 +114,10 @@ stages:
                 runOnce:
                   deploy:
                     steps:
-                      - checkout: azure-sdk-build-tools
-                        path: azure-sdk-build-tools
-                      - template: /tools/generic-blob-upload/publish-blobs.yml@azure-sdk-build-tools
+                      - checkout: self
```

Comment:
Do we need this checkout line anymore if we're not doing multiple repo checkouts?